src/definitions/modules/modal.less

```diff
@@ -479,9 +477,9 @@
   }
 }
 
-/* mini */
-.ui.small.modal > .header:not(.ui) {
-  font-size: @miniHeaderSize;
+/* Tiny */
+.ui.tiny.modal > .header:not(.ui) {
+  font-size: @tinyHeaderSize;
```

Comment:
What does changing this to `tiny` fix?

Reply:
`mini` was defined two times, and `tiny` was missing from the file. This was a typo.

Reply:
Ok cool.